autogen.sh

```diff
@@ -163,6 +163,12 @@ fi
 
 conf_flags="--enable-maintainer-mode --enable-compile-warnings" #--enable-iso-c
 
+echo "MONO_CONC_GC is $MONO_CONC_GC"
+
+if test x$MONO_CONC_GC != x; then
+conf_flags="$conf_flags --with-sgen-default-concurrent"
+fi
+
```

Comment:
We're using an environment variable to make it easy to inject config flags into the build.